applications/sia/templates/deployment.yaml

```diff
@@ -98,6 +98,24 @@ spec:
             - name: "SIA_SENTRY_TRACES_SAMPLE_RATE"
               value: {{ .Values.config.sentryTracesSampleRate | quote }}
             {{- end }}
+            {{- if .Values.config.metrics.enabled }}
+            - name: "KAFKA_BOOTSTRAP_SERVERS"
+              valueFrom:
+                secretKeyRef:
+                  name: "sia-kafka"
+                  key: "bootstrapServers"
+            - name: "KAFKA_SECURITY_PROTOCOL"
+              valueFrom:
+                secretKeyRef:
+                  name: "sia-kafka"
+                  key: "securityProtocol"
+            - name: "KAFKA_CLIENT_CERT_PATH"
+              value: "/etc/sia-kafka/user.crt"
+            - name: "KAFKA_CLIENT_KEY_PATH"
+              value: "/etc/sia-kafka/user.key"
+            - name: "KAFKA_CLUSTER_CA_PATH"
+              value: "/etc/sia-kafka/ca.crt"
+            {{- end }}
```

Comment:
You're also going to need  [volumeMounts](https://github.com/lsst-sqre/phalanx/blob/f15cbcb4b6cfe2536d052b5bb70e5f75e1a2c002/applications/mobu/templates/deployment.yaml#L124-L137) and [volumes](https://github.com/lsst-sqre/phalanx/blob/f15cbcb4b6cfe2536d052b5bb70e5f75e1a2c002/applications/mobu/templates/deployment.yaml#L148-L152) entries for the Kafka keys and certs from the Kafka secret.

Reply:
Oh oops I never added those! Thanks, I've now added them